core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -930,6 +969,22 @@ public static ExceptionVerbosity annotatedServiceExceptionVerbosity() {
         return ANNOTATED_SERVICE_EXCEPTION_VERBOSITY;
     }
 
+    /**
+     * Returns the {@link Predicate} that is used to choose the non-loopback IP v4 address in
+     * {@link SystemInfo#defaultNonLoopbackIpV4Address()}.
+     *
+     * <p>The default value of this flag is {@code null}, which means all valid IPv4 addresses are
+     * preferred. Specify the {@code -Dcom.linecorp.armeria.preferredIpV4Addrs=<csv>} JVM option
+     * to override the default value. The {@code csv} should be
+     * <a href="https://tools.ietf.org/html/rfc4632">Classless Inter-domain Routing(CIDR)</a>s or
+     * exact IP addresses separated by commas.
+     * For example, {@code -Dcom.linecorp.armeria.=211.111.111.111,10.0.0.0/8,192.168.1.0/24}.
```

Comment:
- `...armeria.preferredIpV4Addrs=...`?
- By the way, should we use `Addresses`? :thinking:

Reply:
Yeah, let me use that. 😄